### ISSUE TITLE
To use ```color``` or ```colour```

### ISSUE THREAD

**Issue by epicallan** [devinit/datahub-cms#14]
There are places where we use ```color``` and ```colour``` with in the data hub project. I like color its shorter and its already in lots of places, @dw8547  likes colour and she prefers I refactor and adopt it.
This has become a minor issue due to its use in the data revolution tables. Those tables have a ``color`` column labelled ```colour```. This forced me to write a little extra code to take that in account. I hate writing extra code. 

Can we agree on a standard to adopt throughout the project.

**Comment by akmiller01** [comment]
I would agree with "color" for use in code. The tricky part is that Comms will not accept "color" anywhere public facing, and that could include column names in downloadable datasets.

**Comment by epicallan** [comment]
ohh I hadn't realised column names in downloadable table or in downloadable JSON data would be an issue.

**Comment by akmiller01** [comment]
Might be good to double-check with Comms, but I imagine that unless it's completely unavoidable, they'll want "colour" in anything external.

**Comment by dw8547** [comment]
`color` is a US version of the word and since we are UK based (in terms of HQ and output) we should use `colour` across all products etc. Please let me know where to update.

**Comment by BeckyWillmoth** [comment]
Apologies for the delay, checked with comms and we need to use colour - @epicallan and @akmiller01 can you confirm the impact in terms of time for you both to make this change?
